libs/dh/core/shell/src/lib/dh-core-shell.routes.ts

```diff
@@ -54,13 +52,21 @@ export const dhCoreShellRoutes: Routes = [
         canActivate: [MsalGuard],
       },
       {
-        path: dhMarketParticipantPath,
+        path: 'market-participant',
         loadChildren: () =>
           import('@energinet-datahub/dh/market-participant/shell').then(
             (esModule) => esModule.dhMarketParticipantShellRoutes
           ),
         canActivate: [MsalGuard],
       },
+      {
+        path: 'grid-areas',
+        loadChildren: () =>
```

Comment:
suggestion: there are no sub-routes to `/grid-areas` for now, so let's use the `loadComponent: () => ...` pattern instead.